Userland/Libraries/LibWeb/HTML/NavigableContainer.cpp

```diff
@@ -267,31 +267,33 @@ void NavigableContainer::destroy_the_child_navigable()
     if (navigable->has_been_destroyed())
         return;
     navigable->set_has_been_destroyed();
-    HTML::all_navigables().remove(navigable);
 
     // FIXME: 4. Inform the navigation API about child navigable destruction given navigable.
 
     // 5. Destroy a document and its descendants given navigable's active document.
-    navigable->active_document()->destroy_a_document_and_its_descendants([this] {
+    navigable->active_document()->destroy_a_document_and_its_descendants([this, navigable] {
```

Comment:
What's the purpose of capturing `navigable` here?

Reply:
It is passed to remove() in the callback.

Reply:
Oh duh! The diff viewer confused me :sweat_smile: